python/lsst/qserv/tests/dbLoader.py

```diff
@@ -34,15 +34,19 @@
 import os
 
 from lsst.qserv import css
+from lsst.qserv import qmeta
 from lsst.qserv.admin import nodeAdmin
 from lsst.qserv.admin import nodeMgmt
 from lsst.qserv.wmgr.client import WmgrClient
 
 
 class DbLoader(object):
 
-    def __init__(self, config, data_reader, db_name, multi_node, out_dirname):
-
+    def __init__(self, config, data_reader, db_name, multi_node, out_dirname, czar_list):
+        '''
+        @param czar_list: A list of czar names (like czar1.localdomain) that 
+                          be updated with database schema information.
+        '''
```

Comment:
the pythonistas seem to like their `__init__` docstring to go in the class docstring.